packages/react-router/modules/Router.js

```diff
@@ -39,9 +39,10 @@ class Router extends React.Component {
     }
 
     history.listen(() => {
-      Object.assign(this.router, history)
-      Object.assign(this.router.match, {
-        isExact: history.location.pathname === '/'
+      Object.assign(this.router, history, {
+        match: Object.assign(this.router.match, {
```

Comment:
Oh, sorry. I didn't realize you were assigning to two different objects. Sorry, been a loooooong day for me here in Sweden. The first way you had it was fine 😅